### PR TITLE
[WIP] Document the buildbot setup

### PR DESCRIPTION
**Pull request by bochecha**
This is everything I did to deploy buildbot.ideas-box.org

With this, we have a buildbot which will automatically build Debian packages of Ideascube, every time a new release tag is pushed to Github.

### CODE REVIEW

File: docs/ci-deployment.md [hunk 1]
```diff
@@ -0,0 +1,406 @@
+<p><code># useradd -c &quot;Buildbot user&quot; -d /srv/buildbot -m -s /bin/bash buildbot </code></p>
```
Comment (yohanboniface) [inline docs/ci-deployment.md]: HTML? :)

File: docs/ci-deployment.md [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+<p>Cet utilisateur a aussi besoin des droits de <code>sudo</code> pour certaines commandes. Il es donc n&eacute;cessaire d&#39;ajouter les lignes suivantes avec la commande <code>visudo</code> :</p>
```
Comment (yohanboniface) [inline docs/ci-deployment.md]: French? :)

**Comment by bochecha** [comment]
Sorry about the French/HTML, that was a bad copy-paste from when I had started writing the doc on the BSF wiki, I had just forgotten to remove these lines. :smile: 

By the way, this is not entirely finished, and I once more forgot to add the WIP tag in the title of this PR, which I'm going to fix right now.

I'm definitely happy to receive feedback on what's already here, though.

**Comment by bochecha** [comment]
Given that the doc is getting a bit big, and starts including various files for the deployment, we're moving off to [a dedicated repository](https://github.com/ideascube/ci-conf).

Closing.

**Pull request closed without merging**